redirects/16.0.txt

```diff
@@ -110,6 +110,7 @@ applications/inventory_and_mrp/inventory/shipping/operation.rst applications/inv
 applications/inventory_and_mrp/inventory/shipping/operation/invoicing.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/invoicing.rst                                              # /inventory_and_mrp/inventory/shipping/operation/* -> /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/*
 applications/inventory_and_mrp/inventory/shipping/operation/multipack.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/multipack.rst                                              # /inventory_and_mrp/inventory/shipping/operation/* -> /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/*
 applications/inventory_and_mrp/inventory/shipping/operation/labels.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst                                                    # /inventory_and_mrp/inventory/shipping/operation/* -> /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/*
+applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/labels.rst                                # /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/* -> /inventory_and_mrp/inventory/shipping_receiving/setup_configuration/*
```

Comment:
Small update to the redirect comment, also please move this redirect line below the "O"s to be on line 116 :)

Since a single file is being moved, and not a whole directory, the comment should contain the filename being moved :), whereas commenting `.../folder/*` represents an entire directory move :)
```suggestion
applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/labels.rst                                # /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels -> /inventory_and_mrp/inventory/shipping_receiving/setup_configuration/labels
```